### DISCUSSION TITLE
Error code: 404. Description: Not Found

### DISCUSSION THREAD

**Discussion by Heroeskeeper** [eternnoir/pyTelegramBotAPI#2270] [Q&A]
Hi. I am wondering if someone could please help me.
I have copied and pasted a test async bot as per your docs:

`from telebot.async_telebot import AsyncTeleBot
import asyncio
bot = AsyncTeleBot('TOKEN')

# Handle '/start' and '/help'
@bot.message_handler(commands=['help', 'start'])
async def send_welcome(message):
    await bot.reply_to(message, """\
Hi there, I am EchoBot.
I am here to echo your kind words back to you. Just say anything nice and I'll say the exact same thing to you!\
""")

# Handle all other messages with content_type 'text' (content_types defaults to ['text'])
@bot.message_handler(func=lambda message: True)
async def echo_message(message):
    await bot.reply_to(message, message.text)

asyncio.run(bot.polling())`

I'm getting the following error:

telebot.asyncio_helper.ApiTelegramException: A request to the Telegram API was unsuccessful. Error code: 404. Description: Not Found
Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x000002CDA36CF010>
Unclosed connector
connections: ['[(<aiohttp.client_proto.ResponseHandler object at 0x000002CDA36B1CC0>, 1225146.375)]']
connector: <aiohttp.connector.TCPConnector object at 0x000002CDA36CEE90>

Bot is set up on Telegram and it has been set to admin on a group.
The only difference between the code I posted and my code is the Token which represents my bot's token, obviously.
I am assuming example code in docs would be working examples.

**Comment by Heroeskeeper** [comment]
It appears the code wrapper in the menu bar of the editor is broken too.

**Comment by coder2020official** [comment]
Token is wrong?

**Reply by Heroeskeeper** [reply]
To test that I copied the synchronous telebot sample code and it worked fine with the tame token.

**Reply by coder2020official** [reply]
Version?

**Reply by Heroeskeeper** [reply]
Python? 3.11

**Reply by coder2020official** [reply]
lib

**Reply by Heroeskeeper** [reply]
Probably the latest, installed 2 days ago.
It appears to be working now... what scares me is I have no idea what changed.
Thanks anyway for your time!
